src/server.js

```diff
@@ -274,14 +275,26 @@ const globe = [
 ];
 
 app.get(`${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`, function () {
```

Comment:
Why is there:
```
function() {
  getEvent,
  async (request, response) => { ... }
}
```

Middleware is passed like this: `app.get(<route>, getEvent, (req, res) => { ... })`.

I assume this breaks when you test the app locally, but maybe I'm wrong?

Reply:
I'm generally confused as to what is going on in this method:

It seems like the code looks like this:

```
app.get(<ROUTE>, function () {
    getEvent,
    async (request, response) => {
      // some code
     response.json({ ... })
    }
    // some more code
   // a fetch to the API
   response.json({ ... })
}
`
```
I don't understand this code structure, because it seems like this:
```
getEvent,
(request, response) => { ... }
```
does nothing, since 2 functions are being declared/referenced, but neither is being executed.

I think you want to structure the code like this:
```
app.get(<ROUTE>, getEvent, (request, response) => {
    // call the API with fetch
    fetch(<URL>)
         .then(fetchResponse => {
             // do something w/ response, probably call
            // response.json
         }).catch(err => { 
             // handle error
         })
})
```

Hope this helps! :smile:

Reply:
I'm actually not reviewing my changes. I may have mistakenly request a new review form you?

Reply:
Oh, my bad. My general assumption was commits are fair game to be reviewed, but I'll wait until an explicit request for review in the future.